src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const getUserById = (userId: number) => {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const todoListWithUsers: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(todoListWithUsers);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
```

Comment:
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
  const [isUserError, setIsUserError] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/201521142-830a14ba-ee06-47bc-933c-090c81ab1329.png)
Btw, you can just use an empty string as a default value and change it to error text if it's an error case